main.go

```diff
@@ -102,3 +132,26 @@ func prettyPrint(expectedPrs []PullRequestDetails, fileName string) error {
 	err = t.Execute(f, expectedPrs)
 	return err
 }
+
+func saveIntoFileReleases(releaseList []ReleaseDetails, fileName string) error {
+	fileContents, err := json.MarshalIndent(releaseList, "", "")
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(fileName, fileContents, 0644)
+}
+
+func prettyPrintReleases(releaseList []ReleaseDetails, fileName string) error {
```

Comment:
[nitpick] There's scope of making this and PR print into one method with options. Not mandatory though.

Reply:
Moved the print function to `report-utils.go`, and re-using it.